Jenkinsfile

```diff
@@ -429,7 +435,10 @@ pipeline {
           when {
             beforeAgent true
             anyOf {
-              expression { return params.bindings }
+              allOf {
```

Comment:
Linux and Windows bindings were run in parallel in previous version of a Jenkinsfile. Please, incorporate that logic here